### ISSUE TITLE
Can't start client 18.0.0 (crash during start)

### ISSUE THREAD

**Issue by JCVacher** [ExtensiveTesting/extensivetesting#17]
### description of the issue
When I launch the client 18.0.0, it crashs:

In windows event viewer, I have:
Faulting application name: ExtensiveTestingClient.exe, version: 0.0.0.0, time stamp: 0x5a2e9f14
Faulting module name: ntdll.dll, version: 6.1.7601.23915, time stamp: 0x59b94ee4
Exception code: 0xc0000005
Fault offset: 0x000000000004f23c
Faulting process id: 0x1a78
Faulting application start time: 0x01d3c5c564fa0361
Faulting application path: C:\Program Files\Extensive Testing Client\ExtensiveTestingClient.exe
Faulting module path: C:\Windows\SYSTEM32\ntdll.dll
Report Id: a7be05c1-31b8-11e8-b0b7-847beb26756d

### steps to reproduce the issue
install client 18.0.0 (on a windows 7 professional), and launch the Client

### additionnals informations like versions, logs, screenshots
As ask in the google group, I try with a version 19.0.0alpha version. I have the same crash. Logs are in the attached zip

[Logs_ExtensiveTestingClient_19.0.0alpha1_64bit_Portable.zip](https://github.com/ExtensiveTesting/extensivetesting/files/1870978/Logs_ExtensiveTestingClient_19.0.0alpha1_64bit_Portable.zip)

**Comment by dmachard** [comment]
Have you Qt or PyQT or Python already installed on your environment ?
If yes which versions ?

**Comment by JCVacher** [comment]
Yes, I have Python 2.7, and PyQt (PyQt GPL v4.9.6) already installed.
I think there were installed by the 1st client version I installed on this server (I don't remember the version... it was in September 2016)

**Comment by dmachard** [comment]
The new client 18.0.0 is compilated with the version major version Qt5 and the wrapper PyQt 5.9.1
I think the client loads the wrong dll because of the pyqt 4.9.6 on your system.

**Comment by dmachard** [comment]
Can you retry without pyqt 4.9.6 installed on your system ?

**Comment by JCVacher** [comment]
Sorry, yesterday afternoon I hadn't time to do the test...
I do the test this morning: I uninstall pyqt 4.9.6 (restart my laptop to be sure :-) )
But I still have the crash... (But I don't find anymore the error in windows event viewer...)

[Logs.zip](https://github.com/ExtensiveTesting/extensivetesting/files/1874598/Logs.zip)

**Comment by dmachard** [comment]
Thanks for logs, I uploaded a new debug version with more logs if you can retry again with this version
https://www.extensivetesting.org/ftp/.alpha/ExtensiveTestingClient_19.0.0alpha2_64bit_Portable.zip

By the way, it's always possible to compile the client with qt4 directly from source.
But by default, I provide no more support with this version, only for qt5.

**Comment by JCVacher** [comment]
I still have the crash...
I attach logs.

[Logs_19.0.0alpha2.zip](https://github.com/ExtensiveTesting/extensivetesting/files/1875158/Logs_19.0.0alpha2.zip)

NB: it's very strange: I test on an other laptop with windows 7 professional
 (where previous version of the client were installed as on mine) and there is no problem: the client runs... I'll try to find what could the difference between this 2 laptops...

**Comment by JCVacher** [comment]
I've read that some 64bits dlls are installed in system32... then I thinks it's not the problem.

I continue to do some tests: I've remove all pythons installation, but I still have the same crash...

I continue to search, and tell you if I find something...
